src/graph/type_indexededgelist.c

```diff
@@ -1691,3 +1691,62 @@ int igraph_incident(const igraph_t *graph, igraph_vector_t *eids,
 
     return 0;
 }
+
+
+/**
+ * \function igraph_is_same_graph
+ * \brief Check if two graphs are topologically the same
+ *
+ * \param graph1 The first graph object.
+ * \param graph2 The second graph object.
+ * \return 1 if they are the same, 0 if they are different.
+ *
+ * NOTE: this function is mostly for internal use in unit tests.
```

Comment:
Is there any good reason for this? If the function is robust, its use should not be discouraged (and it _should_ be robust). Testing is the most common, but by far not the only use case.

There are several other changes I want to make to the docs. Is it okay if I make edits to this text on this branch?